File: src/main/java/com/curseclient/mixin/accessor/entity/AccessorPlayerControllerMP.java
package com.curseclient.mixin.accessor.entity;

import net.minecraft.client.multiplayer.PlayerControllerMP;
import org.spongepowered.asm.mixin.Mixin;
import org.spongepowered.asm.mixin.gen.Accessor;

@Mixin(PlayerControllerMP.class)
public interface AccessorPlayerControllerMP {
    @Accessor float getCurBlockDamageMP();
    @Accessor void setCurBlockDamageMP(float value);
    @Accessor void setBlockHitDelay(int blockHitDelay);
    @Accessor void setCurrentPlayerItem(int currentPlayerItem);
}

File: src/main/kotlin/com/curseclient/mixin/accessor/entity/AccessorPlayerControllerMP.kt
package com.curseclient.mixin.accessor.entity

import net.minecraft.client.multiplayer.PlayerControllerMP
import org.spongepowered.asm.mixin.Mixin
import org.spongepowered.asm.mixin.gen.Accessor

@Mixin(PlayerControllerMP::class)
interface AccessorPlayerControllerMP {
    @get:Accessor("curBlockDamageMP")
    @set:Accessor("curBlockDamageMP")
    var curBlockDamageMP : Float
}